### DISCUSSION TITLE
tau-u

### DISCUSSION THREAD

**Discussion by sunflowerecnu** [jazznbass/scan#143] [Q&A] [answered]
one question about D  in  "A vs. B - TrendA".
If tau_method="a",  Ds from "parker" and "complete" are different.
If tau_method="b",  Ds from the two methods are the same.
Why?

try <- scdf(
  c(A = 40, 40, 40, 40, 30, 30, 
    B = 40, 50, 50, 60, 60, 70, 80, 70, 80, 90, 90, 90)
  )

tau_u(try, method="complete",tau_method="a") %>% print(complete = TRUE)
tau_u(try, method="parker",tau_method="a") %>% print(complete = TRUE)
#results: D=87  D=72

tau_u(try, method="complete",tau_method="b") %>% print(complete = TRUE)
tau_u(try, method="parker",tau_method="b") %>% print(complete = TRUE)
#results: D=108.37

**Comment by jazznbass** [comment, marked as answer]
The "parker" method is not defined for Tau B (with ties). I do not think it is sensible to transfer the algorithm describe by Parker to a context with ties.
So both approaches are identical for Tau B

**Reply by sunflowerecnu** [reply]
Thanks a lot.
